packages/dds/tree/src/feature-libraries/chunked-forest/codec/formatGeneric.ts

```diff
@@ -42,8 +43,11 @@ const EncodedFieldBatchBase = Type.Object(
  * @param version - format version. Must be changed if there is any change to the generic schema, or the `shape` schema.
  * @param shape - schema for union of shape format, see {@link DiscriminatedUnionDispatcher}.
  */
-export const EncodedFieldBatchGeneric = <TShapeSchema extends TSchema>(
-	version: number,
+export const EncodedFieldBatchGeneric = <
+	TShapeSchema extends TSchema,
+	TVersion extends FieldBatchFormatVersion,
```

Comment:
Added this type to change the type of `version` from `number` to `FieldBatchFormatVersion`